app/controllers/mentor/configure_controller.rb

```diff
@@ -2,6 +2,7 @@ class Mentor::ConfigureController < ApplicationController
   before_action :authenticate_user!
 
   def show
+    @mentored_track_ids = Set.new(current_user.mentored_tracks.pluck(:id))
```

Comment:
What's the reason behind using `Set.new` here rather than leaving it as an array?